services/api/handlers/projects.js

```diff
@@ -49,6 +49,25 @@ exports.post = {
 };
 
 exports.get = {
+  oneById: function(request, reply) {
+    request.server.methods.projects.findOneById(
+      request.params.project,
```

Comment:
yup, it's passing it directly into the pg query function, which expects an array